app/lib/actions/katello/content_view/promote.rb

```diff
@@ -2,6 +2,11 @@ module Actions
   module Katello
     module ContentView
       class Promote < Actions::EntryAction
+        extend ApipieDSL::Class
+        include ::Actions::ObservableAction
+        include Actions::Katello::JailConcern::Organization
+        include Actions::Katello::JailConcern::ContentView
```

Comment:
Just for the record, this does not work atm, organization jail is overwritten in the include block by content view.

Reply:
Since the Jail.allow method concatenates to a list, you can call it multiple times. So I suggest you assume Jail already exists, move these includes down after Jail class definition and append from the included block.

Reply:
I've tried that one, but when I try something like `base::Jail` or `base.const_get(:Jail)` it returns an object that doesn't understand `.inspect` even.